valhalla/odin/narrativebuilder.h

```diff
@@ -360,6 +360,35 @@ class NarrativeBuilder {
   /////////////////////////////////////////////////////////////////////////////
   std::string FormVerbalPostTransitionTransitInstruction(Maneuver& maneuver);
 
+  /////////////////////////////////////////////////////////////////////////////
+  std::string FormVerbalSuccinctStartTransitionInstruction(Maneuver& maneuver);
+
+  std::string FormVerbalSuccinctDestinationTransitionInstruction();
```

Comment:
remove for now

Reply:
I'm going to leave for now since we discussed this.